01-cloudformation/README.md

```diff
@@ -160,19 +160,34 @@ name.
 
 _Why do we prefer the YAML format for CFN templates?_
 
+#### The intrinsic functions
+
 #### Question: Protecting Resources
 
 _What else can you do to prevent resources in a stack from being deleted?_
 
 See [DeletionPolicy](https://aws.amazon.com/premiumsupport/knowledge-center/cloudformation-accidental-updates/).
 
+#### Use IAM to restrict users and utilize the DeletionPolicy attribute
+
 _How is that different from applying Termination Protection?_
 
+#### It not only prevents deletion but also prevents updates
```

Comment:
These methods are different from applying Termination Protection in that they use IAM policies to prohibit or limit deletions.